### PR TITLE
Update to READMEs.

### PR DESCRIPTION
**Pull request by jj-jitterbit**

### CODE REVIEW

File: charts/agent-group/README.md [hunk 1]
```diff
@@ -1,60 +1,66 @@
 # agent-group
 
-[agent-group](https://github.com/jitterbit/charts) is a group of agents in the Harmony platform.
+The [agent-group](https://github.com/jitterbit/charts) is a group of Jitterbit Private Agents of the [Jitterbit
```
Comment (lbrown223) [inline charts/agent-group/README.md]:
```suggestion
The [agent-group](https://github.com/jitterbit/charts) is a group of Private Agents of the [Jitterbit
```

File: charts/agent-group/README.md [hunk 2]
```diff
@@ -1,60 +1,66 @@
...
+## Installing or Upgrading the Chart
+
+To install or upgrade the chart with the release name `my-release`, use the command:
 
 ```bash
 $ helm upgrade --install --namespace apps --values ./my-values.yaml my-release jitterbit/agent-group
 ```
+This example command deploys the `agent-group` on the Kubernetes cluster in the `apps` namespace.
+
+The [configuration](#configuration) section lists common parameters that will be configured often during
```
Comment (lbrown223) [inline charts/agent-group/README.md]:
```suggestion
The [configuration](#configuration) section lists common parameters that will be configured during
```

File: charts/agent-group/README.md [hunk 3]
```diff
@@ -1,60 +1,66 @@
...
+installation and upgrade.
 
-The command deploys `agent-group` on the Kubernetes cluster in the `apps` namespace.
-The [configuration](#configuration) section lists common parameters that will be configured often
-during installation and upgrade.
 For a full list of the parameters that can be configured, see the [values.yaml](values.yaml) file.
 
-> **Tip**: List all releases using `helm list`
+> **Tip**: To list all releases, use the command `helm list`
```
Comment (lbrown223) [inline charts/agent-group/README.md]:
```suggestion
**Tip**: To list all releases, use the command `helm list`.
```

File: charts/agent-group/README.md [hunk 4]
```diff
@@ -1,60 +1,66 @@
...
-`image.digest` | The image digest to pull | `nil` You must provide a digest or a tag
-`agent.registerjson` | The agent group's register.json file | `nil`
-`agent.jitterbitconf` | The agent group's jitterbit.conf file | `{}`
-`hpa.enable` | Enable/disable the Horizontal Pod Autoscaler | `true`
-`hpa.minReplicas` | If `hpa.enabled` is `true`, set minimum number of replicas | `1`
-`hpa.maxReplicas` | If `hpa.enabled` is `true`, set maximum number of replicas | `1`
-`hpa.metrics` | If `hpa.enabled` is `true`, scale based on these metrics | `{...}` See the [values.yaml](values.yaml) file
-`replicas` | If `hpa.enabled` is `false` or the Horizontal Pod Autoscaler is malfunctioning, set the static number of replicas | `1`
-`resources` | Resources allocated for the container | `{requests: {cpu: 50m, memory: 0.75Gi}, limits: {cpu: 250m, memory: 1Gi}}`
+This table lists commonly-configured parameters of the agent-group chart and their default values.
```
Comment (lbrown223) [inline charts/agent-group/README.md]:
```suggestion
This table lists commonly configured parameters of the agent-group chart and their default values.
```

File: README.md [hunk 5]
```diff
@@ -4,32 +4,30 @@
...
-## Install a Jitterbit API Gateway
+## Installing a Jitterbit Private Agent Group
 
 ```bash
-helm upgrade --install api-gateway jitterbit/api-gateway
+helm upgrade --install agent-group jitterbit/agent-group
 ```
 
-For more details on installing the Jitterbit API Gateway please see the [chart's README](https://github.com/jitterbit/charts/tree/master/charts/api-gateway).
+For details on installing a Jitterbit Private Agent Group, see the `agent-group` chart's
```
Comment (lbrown223) [inline README.md]:
```suggestion
For details on installing a Jitterbit Harmony Private Agent Group, see the `agent-group` chart's
```

**Review by lbrown223 — DISMISSED** [review]
Approved with minor suggestions

**Review by sean-krail — DISMISSED** [review]

File: charts/agent-group/README.md (hunk 1 above)
Comment (jj-jitterbit) [reply charts/agent-group/README.md]: Fixed.

File: charts/agent-group/README.md (hunk 2 above)
Comment (jj-jitterbit) [reply charts/agent-group/README.md]: Fixed.

File: charts/agent-group/README.md (hunk 3 above)
Comment (jj-jitterbit) [reply charts/agent-group/README.md]: Fixed.

File: charts/agent-group/README.md (hunk 4 above)
Comment (jj-jitterbit) [reply charts/agent-group/README.md]: Fixed.

**Review by lbrown223 — APPROVED** [review]

**Review by sean-krail — APPROVED** [review]

**Pull request merged**